test/integration/src/main/resources/META-INF/services/io.pravega.auth.AuthHandler

```diff
@@ -0,0 +1,10 @@
+#
```

Comment:
Are these files needed?

Reply:
Yes, these files are necessary for the custom test auth handler to get loaded via the service loader.